src/components/PersonalSettings.vue

```diff
@@ -72,24 +83,25 @@ export default {
 			oauthConnectionErrorMessage: loadState('integration_openproject', 'oauth-connection-error-message'),
 			oauthConnectionResult: loadState('integration_openproject', 'oauth-connection-result'),
 			userSettingDescription: USER_SETTINGS,
-			authMethods: AUTH_METHOD,
 			errorMessages,
 		}
 	},
 	computed: {
 		connected() {
 			if (!this.state.admin_config_ok) return false
-			return this.state.token && this.state.token !== ''
-				&& this.state.user_name && this.state.user_name !== ''
+			return !!this.state.token && !!this.state.user_name
+		},
+		isOIDCMethod() {
```

Comment:
```suggestion
		isAuthMethodOIDC() {
```
or
```suggestion
		isAuthenticatedByOIDC() {
```

and similar for `isOauthMethod`, those names did confuse me

Reply:
I renamed them to `isOIDCAuthMethod` and `isOauth2AuthMethod`